components/esp_timer/src/esp_timer.c

```diff
@@ -325,8 +319,8 @@ esp_err_t esp_timer_delete(esp_timer_handle_t timer)
         timer->period = 0;
         err = timer_insert(timer, false);
     }
-    timer_list_unlock(ESP_TIMER_TASK);
-    return err;
+	irq_unlock(key);
```

Comment:
can we please use the spaces instead of tabs.

Reply:
of course, I did for everything but missed this one. tks.